app-config.yaml

```diff
@@ -99,6 +99,16 @@ proxy:
         X-Atlassian-Token: 'no-check'
         User-Agent: ${JIRA_USER_AGENT}
 
+    # Plugin: ACR
+    '/acr/api':
+      target: ${ACR_URL}
+      changeOrigin: true
+      headers:
+        # If you use Bearer Token for authorization, please replace the 'Basic' with 'Bearer' in the following line.
+        Authorization: 'Basic ${ACR_AUTH_TOKEN}'
```

Comment:
sorry my bad, would be good to keep it as `Bearer` over `Basic`

Reply:
fixed. Please use the `suggestion` festure when suggesting one-line changes.